driver-legacy/src/main/com/mongodb/DBCursor.java

```diff
@@ -616,6 +618,7 @@ public int length() {
      * @throws MongoException if failed
      * @see #count()
      */
+    @VisibleForTesting(otherwise = PRIVATE)
```

Comment:
I'm not convinced about this one.  DBCursor is a public class and this method has been part of the API forever.  I think of VisibleForTesting as an annotation we apply to driver-internal classes as a way to signal to _us_ the intent, not something that we want consumers of our API to be aware of.  AFAIK this would be the first and only API element to be annotated as such, so we're be setting a precedent here that I don't think we'd ever want to repeat.

Reply:
Yes, I apologize. `VisibleForTesting` is indeed internal, and it is weird to add it to public API program elements.

Done.